content/_index.md

```diff
@@ -10,36 +10,35 @@ disable_toc: true
 
 # Welcome to Datadog Docs!
 
-If you're new here, read below for the 10,000 foot view of Datadog.
+If you're new here, read below for the high-level view of Datadog.
 
 {{< partial name="tile-nav/tile-nav.html" >}}
 
 Your application stack is teeming with unused metadata that's trying to tell a story: things aren't humming along as well as they should be.
 
-Exception thrown; Database queries slowing; Cache misses rising; Upstream service flapping; Error log growing. Each of these chatters a part of the story, but they're hard to make sense of—or act on meaningfully—when taken separately from the others.
+Exceptions thrown; database queries slowing; cache misses rising; upstream services flapping; error logs growing. Each of these tells a part of the story, but it's hard to make sense of—or act on meaningfully—when taken separately from the others.
```

Comment:
Using semi-colons to break up this list is sort of weird imho, but capitalising each item was even weirder. 😉